test/experimental_apis/rdf.js

```diff
@@ -356,6 +356,7 @@ describe('Experimental RDF emulation', () => {
                                                 h.createRandomSecKey(),
                                                 h.createRandomNonce());
     const notMyRdf = notMyMd.emulateAs('rdf');
-    return should(notMyRdf.nowOrWhenFetched(null, encrypted)).be.rejectedWith('Core error: Symmetric decryption failed');
+    return should(notMyRdf.nowOrWhenFetched(null, encrypted))
```

Comment:
@bochaco I'm unsure of this test.

The changes you made above, avoid decryption if there's no value, as keys were changed due to rejection of an app (as I understand).

Here you're simply attempting to decrypt with bad encryption keys... Which _should_ yield the previous error, and not 'no id', or?